### PR TITLE
[WEB-9140] Migrate serato/sws-js-sdk to Github Actions

### PR DESCRIPTION
**Pull request by sameeraroshan-serato**
1. Added GitHub action to build the library
2. Deleted the .`travis.yml` file

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        node-version: [10.x]
+
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
```
Comment (AnnieCSerato) [inline .github/workflows/build.yml:24]: Doesn't this github action already have options for caching?

**Review by AnnieCSerato — COMMENTED** [review]
Looking good, got a couple improvement comments

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+name: Build and test sws-js-sdk
+
+on:
+  push:
+    branches:
+      - master
+      - "/^v\\d+\\.\\d+(\\.\\d+)?(-\\S*)?$/"
```
[the remark this answers is not in the document]
Comment (sameeraroshan-serato) [reply .github/workflows/build.yml:7]: Just copied from old travis file.It suppose to check release versions like v1.2 , v1.2.3, Anyway ill add it as a comment.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x]
```
[the remark this answers is not in the document]
Comment (sameeraroshan-serato) [reply .github/workflows/build.yml:15]: There is a nvm file which is v16.16.0 but travis build used v10.

File: .github/workflows/build.yml (hunk 1 above)
Comment (sameeraroshan-serato) [reply .github/workflows/build.yml:24]: Actually it does.But form their [documentation](https://github.com/actions/setup-node), it caches only the global package data. This action does not cache the project specific `node_modules` directory. But I can add   `cache: 'npm'` to cache global packages.

File: .github/workflows/build.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (aosmakserato) [reply .github/workflows/build.yml:18]: Have you tried running v16 on Github actions? It is better to use the newer version if it does not break anything.

[the remark this answers is not in the document]
Comment (sameeraroshan-serato) [reply .github/workflows/build.yml:18]: I'll give it a try.

[the remark this answers is not in the document]
Comment (sameeraroshan-serato) [reply .github/workflows/build.yml]: Added the `.nvmrc` based node setup.

**Review by aosmakserato — APPROVED** [review]

**Review by AnnieCSerato — APPROVED** [review]

**Pull request merged**